tests/ttnn/unit_tests/gtests/ccl/kernels/fabric_erisc_datamover_sender_worker_sender.cpp

```diff
@@ -145,62 +156,40 @@ void kernel_main() {
         uint64_t buffer_address = sender.edm_buffer_addr +
                                   (*sender.buffer_index_ptr * (sender.buffer_size_bytes + sizeof(eth_channel_sync_t)));
         sender.send_payload_blocking_from_address(packet_addr, packet_size);
+        DPRINT << "noc write barrier\n";
         noc_async_writes_flushed();
+        DPRINT << "cb pop front\n";
         cb_pop_front(cb_id_in0, pages_to_send);
+        DPRINT << "cb pop front done\n";
     }
 
+    DPRINT << "DONE MAIN LOOP\n";
     if constexpr (!mcast_mode) {
+        DPRINT << "TEARDOWN\n";
         sender.wait_for_empty_write_slot();
 
         auto& packet_header = *reinterpret_cast<tt::fabric::PacketHeader*>(a_packet_header_addr);
         ASSERT(*last_message_semaphore_address == 0);
         packet_header.reserved = 0xE;
         packet_header.reserved2 = 0xFFFF;
         packet_header.to_atomic_inc();
-        packet_header.to_chip_unicast(tt::fabric::UnicastRoutingCommandHeader{1});
+        packet_header.to_chip_unicast(tt::fabric::UnicastRoutingCommandHeader{2});
```

Comment:
I think it would be good to turn the 2 and the 0xE into an external constant just to aid in the self-documenting aspect since it is not clear why those magic numbers were chosen. Either that or add a comment perhaps?